### DISCUSSION TITLE
can we add lights in our flare engine mod?

### DISCUSSION THREAD

**Discussion by BuzzLEquerre** [flareteam/flare-game#962] [General]
I would like to create a lighting system to have torches etc in the dungeons or lamps in the villages at night, is this possible?

**Comment by dorkster** [comment]
There is no dynamic lighting system in Flare, so the best you can do is create fake "lights" as part of the tileset.

Here's a quick before/after mock-up of what that could potentially look like:
![Screenshot_20240809_133058](https://github.com/user-attachments/assets/b14d8a71-d515-43be-b03c-ecc2a1034c99)

**Comment by BuzzLEquerre** [comment]
ok so I'm going to create lights in all possible directions and so I could do "lighting".
Thank you very much for your advice! 
And is it possible to make objects transparent when the player is hidden behind them?

**Reply by dorkster** [reply]
>And is it possible to make objects transparent when the player is hidden behind them?

Not at the moment. We do compensate for this a little by drawing an indicator above the player (and other entities) when they are partially covered by a tile that's as tall as the entity it is obscuring.

**Reply by BuzzLEquerre** [reply]
Okay, it doesn't matter, you've already done a fantastic job, I know nothing about coding, etc., I can move around in my world! I admire your work, it allows people like me who know nothing about it to still be able to realize their dreams :)
